### PR TITLE
Create norway_air.markdown

### PR DESCRIPTION
**Pull request by Danielhiversen**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#20683

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/norway_air.markdown [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+sharing: true
+footer: true
+logo: metno.png
+ha_category: Health
+ha_iot_class: "Cloud Polling"
+ha_release: 0.88
+---
+
+
+The `norway_air` component [queries](https://luftkvalitet.miljostatus.no/) the Noreay air quality [data feed](https://api.met.no/weatherapi/airqualityforecast/0.1/documentation) provided by the Norwegian Meteorological Institute.
```
Comment (DubhAd) [inline source/_components/norway_air.markdown]: Noreay ?

I'm guessing that's a typo ;)

**Review by DubhAd — DISMISSED** [review]

File: source/_components/norway_air.markdown (hunk 1 above)
Comment (Danielhiversen) [reply source/_components/norway_air.markdown]: oi :/

**Comment by klaasnicolaas** [comment]
Why closing this PR?

**Comment by Danielhiversen** [comment]
I will refactor the python code, so I reopen it later

**Review by frenck — APPROVED** [review]

**Pull request merged**